cypress/page/shoping-cart.page.ts

```diff
@@ -0,0 +1,13 @@
+class ShoppingCartPage {
+  private proceedToCheckoutBtn: string;
+
+  constructor() {
+    this.proceedToCheckoutBtn = ".cart_navigation span";
+  }
+
+  public clickProceedToCheckout(): void {
+    cy.get(this.proceedToCheckoutBtn).click();
+  }
+}
+
+export default new ShoppingCartPage();
```

Comment:
not sure about this approach, I prefer the one suggested by the guide, export the class and then have a page/index.ts so you can import from the page index in the buy-shirt.cy.ts file